app/api/private-cloud/products/[licencePlate]/comments/_operations/list.ts

```diff
@@ -1,18 +1,29 @@
 import prisma from '@/core/prisma';
 
-export async function listOp(licencePlate: string) {
-  const comments = await prisma.privateCloudComment.findMany({
-    where: {
-      project: {
-        licencePlate: licencePlate,
+export async function listOp(licencePlate: string, requestId?: string) {
```

Comment:
let's fetch `product` by `licencePlate` first and generate the query dynamically if `requestId` is provided.